keyboards/leafcutterlabs/bigknob/readme.md

```diff
@@ -0,0 +1,15 @@
+# bigknob
+
+A 4 key Macropad with Rotatry Encoder based on Atmega32u4.
+
+![](https://github.com/LeafCutterLabs/bigKNOB/blob/master/Media/thinkpad.jpg)
```

Comment:
Please rehost this on imgur and add `l` to the end of the filename to make it thumbnail sized.